tests/selection_sort_tests.py

```diff
@@ -0,0 +1,48 @@
+# Import hacks for compatibility
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
Why is this here?

Reply:
Tests are in a test folder, nested one layer deep from the actual algorithms. This was a way @IvanIsCoding showed me to get the parent in the most safe way, irrespective of where you are running the program from.